packages/glade.rb

```diff
@@ -22,15 +22,8 @@ class Glade < Package
   depends_on 'pygobject'
   depends_on 'graphite'
 
-  def self.build
-    ENV['CFLAGS'] = '-fuse-ld=lld'
-    ENV['CXXFLAGS'] = '-fuse-ld=lld'
-    system "meson setup #{CREW_MESON_FNO_LTO_OPTIONS} -Dgtk_doc=false -Dman=false build"
-    system 'meson configure build'
-    system 'ninja -C build'
-  end
+  gnome
 
-  def self.install
-    system "DESTDIR=#{CREW_DEST_DIR} ninja -C build install"
-  end
+  pre_meson_options "CFLAGS='-fuse-ld=lld' CXXFLAGS='-fuse-ld=lld'"
+  meson_options '-Dgtk_doc=false -Dman=false build'
```

Comment:
shouldn't this be `meson_options '-Dgtk_doc=false -Dman=false'`?

Reply:
Yes.